### DISCUSSION TITLE
Outdated documentation

### DISCUSSION THREAD

**Discussion by Naatan** [MontFerret/ferret#682] [Q&A]
https://www.montferret.dev/docs/getting-started/

This talks about `--cdp` and `--cdp-launch` as a ways to use browsers, but these flags are not available on the latest version.

Aside from this mention I was not able to find any documentation on how to use browsers.

**Comment by ziflex** [comment]
Hey, sorry for that inconvenience. I'll update the docs soon. But for now, you can find info on how to use CLI [here](https://github.com/MontFerret/cli).

The documentation is a pretty weak spot of the project, unfortunately.
How to work with browser you can find in [this blog post](https://www.montferret.dev/blog/say-hello-to-ferret/).
